components/org.wso2.carbon.identity.remotefetch.core.ui/src/main/java/org/wso2/carbon/identity/remotefetch/core/ui/client/RemoteFetchConfigurationClient.java

```diff
@@ -153,6 +153,7 @@ private static RemoteFetchConfiguration parseJsonToConfiguration(String jsonObje
 
         Gson gson = new Gson();
         RemoteFetchConfiguration fetchConfiguration = gson.fromJson(jsonObject, RemoteFetchConfiguration.class);
+        fetchConfiguration.setRemoteResourceURI(fetchConfiguration.getRepositoryManagerType());
```

Comment:
It looks wrong to set Remote Resource URI as a Repository manager type.
Looks still the method name does not correspond to what it supposed to do